src/main/java/io/pinecone/commons/IndexInterface.java

```diff
@@ -697,4 +697,63 @@ W update(String id, List<Float> values, Struct metadata, String namespace, List<
      * @return A generic type {@code Y} that contains the stats about the index.
      */
     Y describeIndexStats(Struct filter);
+
+    /**
+     * Retrieves up to 100 vector IDs from a given namespace.
+     *
+     * @param namespace The namespace that holds the vector IDs you want to retrieve.
+     * @return A generic type {@code Y} that contains vector IDs.
+     */
+    Z list(String namespace);
+
+    /**
+     * Retrieves up to `n` vector IDs from a given namespace, where `limit` == `n`.
```

Comment:
I don't think the backticks render in the docs output. You can double check by doing `gradle generateJavadoc` and then opening the `index.html` file in the docs folder yourself.

I've been using the `{@code n}` format myself, but it's up to you.

Reply:
Ah ty!